### PR TITLE
Mongo support

### PR DESCRIPTION
**Pull request by hghianni**

### CODE REVIEW

**Comment by hghianni** [comment]
Hi @arcuri82, I updated all the controllers with the new method getMongoConnection which is used to get the MongoClient and insert data into db. With this change, gestaohospital should now show an improvement in the experiments. @jgaleotti and I were able to reproduce errors and we are working now on finding solutions. Thanks!

File: jdk_8_maven/cs/rest-gui/gestaohospital/src/main/java/br/com/codenation/hospital/resource/HospitalResource.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 @RestController
 @RequestMapping(path = Constant.V1)
 public class HospitalResource {
-	private static final Logger LOGGER = LoggerFactory.getLogger(ProductResource.class);
+	private static final Logger LOGGER = LoggerFactory.getLogger(HospitalResource.class);
```
Comment (arcuri82) [inline jdk_8_maven/cs/rest-gui/gestaohospital/src/main/java/br/com/codenation/hospital/resource/HospitalResource.java:27]: why this? if it was a bug in the SUT, we should leave it as it is

File: jdk_8_maven/cs/rest-gui/gestaohospital/src/main/java/br/com/codenation/hospital/resource/PatientResource.java [hunk 2]
```diff
@@ -26,7 +26,7 @@
 @RestController
 @RequestMapping(path = Constant.V1Path)
 public class PatientResource {
-	private static final Logger LOGGER = LoggerFactory.getLogger(ProductResource.class);
+	private static final Logger LOGGER = LoggerFactory.getLogger(PatientResource.class);
```
Comment (arcuri82) [inline jdk_8_maven/cs/rest-gui/gestaohospital/src/main/java/br/com/codenation/hospital/resource/PatientResource.java:29]: why this? if it was a bug in the SUT, we should leave it as it is

**Review by arcuri82 — CHANGES_REQUESTED** [review]

**Comment by arcuri82** [comment]
@hghianni thx ;) see minor comments.

**Pull request merged**